pinot-common/src/main/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcherFactory.java

```diff
@@ -42,14 +42,15 @@ public static SegmentFetcherFactory getInstance() {
   }
 
   public static final String PROTOCOLS_KEY = "protocols";
-  public static final List<String> DEFAULT_PROTOCOLS = Collections.unmodifiableList(Arrays.asList("file", "http"));
+  public static final List<String> DEFAULT_PROTOCOLS = Collections.unmodifiableList(Arrays.asList("file", "http", "multipart"));
```

Comment:
not sure I understand the difference between "http" and "multipart". Should we call it multipart-http? Is the idea of introducing a multi-part fetcher to enable fetchign multiple segments in one go?

Reply:
I see, what would you suggest? Not sure I can think of a better one.